_episodes/07-thresholding.md

```diff
@@ -306,7 +306,7 @@ we have seen before in
 [the _Image Representation in skimage_ episode]({{ page.root }}{% link _episodes/03-skimage-images.md %}).
 
 ~~~
-image = skimage.io.imread(fname="data/maize-root-cluster.jpg")
+image = iio.imread("data/maize-root-cluster.jpg")
```

Comment:
So far in this episode, the filename has been passed as a positional argument, so I find it more consistent to keep it this way in the rest of the episode.

Wait, in episode 03, it says that we will always use named arguments. Let me update this, then.

Reply:
Fixed in b11b829ea.